scripts/build/Platform/Windows/build_config.json

```diff
@@ -175,6 +175,28 @@
       "TEST_SCREENSHOTS": "True"
     }
   },
+  "benchmark_test_gpu_profile": {
```

Comment:
We'd eventually want to reuse the artifacts from the initial windows profile gpu test build job to save time and resources, but we can improve on that later. See the `profile_pipe` section to see how that's done. It basically calls each job in an ordered list.

Reply:
we can revisit this later